pulp_ansible/app/viewsets.py

```diff
@@ -385,25 +385,13 @@ class CollectionRemoteViewSet(RemoteViewSet):
     queryset = CollectionRemote.objects.all()
     serializer_class = CollectionRemoteSerializer
 
-    @extend_schema(
-        description="Trigger an asynchronous update task",
-        responses={202: AsyncOperationResponseSerializer},
-    )
-    def update(self, request, pk, **kwargs):
-        """Update remote."""
-        partial = kwargs.pop("partial", False)
-        lock = [self.get_object()]
+    def async_reserved_resources(self, instance):
```

Comment:
This is very cool, I did not know this existed. Does this mean it will lock all those repositories on delete as well? (does delete lock on the instance?)

Reply:
That's a good question. Need to check.

Reply:
https://github.com/pulp/pulpcore/blob/main/pulpcore/app/viewsets/base.py#L543

Yes, it does.